test/item_test.rb

```diff
@@ -17,5 +26,25 @@ def test_item_has_a_name
 
     assert_equal 'Item Qui Esse', item.name
   end
+
+  def test_find_invoice_items_returns_invoice_items_associated_with_item
+    result_1 = @item_repository.items[0].invoice_items
+    result_2 = @item_repository.items[9].invoice_items
+
+    assert_equal 0, result_1.count
+    assert_equal 1, result_2.count
+  end
+
+  def test_find_merchant_returns_a_merchant_associated_with_item
+    result_1 = @item_repository.items[0].merchant
+
+    assert_equal 'Schroeder-Jerde', result_1.name
+  end
+
+  private
+
+  def fixtures_directory
+    File.expand_path('../../data/fixtures', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.